### DISCUSSION TITLE
Treeview in Reports

### DISCUSSION THREAD

**Discussion by TomAff** [moneymanagerex/moneymanagerex#3852] [Q&A]
Hello,

is there a possibility to use a treeview in reports like this
![2021-09-10 08_19_54-Window](https://user-images.githubusercontent.com/46572503/132809432-9a494336-94e4-4191-b463-2f1c480ce058.jpg)

**Comment by vomikan** [comment]
FYI

https://stackoverflow.com/questions/16926752/expand-collapse-table-rows-with-jquery

**Comment by TomAff** [comment]
It's not easy. You didn't have any report using this?

**Reply by vomikan** [reply]
Hide/Unhide for frames is using on Home Page.
Nothing for reports.

**Reply by TomAff** [reply]
maybe it's easiest to integrate PHP after all :)

**Reply by vomikan** [reply]
That is too hard :)
